shotover-proxy/tests/runner/hotreload_redis_tests.rs

```diff
@@ -0,0 +1,100 @@
+use crate::shotover_process;
+use redis::{Client, Commands};
+use std::time::Duration;
+use test_helpers::docker_compose::docker_compose;
+use tokio::time::sleep;
+
+#[tokio::test]
+async fn test_hotreload_basic_redis_connection() {
+    println!(" Test 1: Basic Redis connection with hotreload");
+    let _compose = docker_compose("tests/transforms/docker-compose-redis.yaml");
+    sleep(Duration::from_millis(2000)).await;
+    println!(" Starting shotover with --hotreload...");
```

Comment:
Please remove all the `println`s.
If the test has failed, then we are able to tell how far we got through the test by looking at the stacktrace of the panic.